### PR TITLE
Fixed Inline block edit identifier validation

### PR DESCRIPTION
**Pull request by niravkrish**
### Description (*)

Dot is not allowed when editing CMS block in-line

### Fixed Issues (if relevant)

1. magento/magento2#21374: Dot is not allowed when editing CMS block in-line

### Manual testing scenarios (*)

1. Manual Testing

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @niravkrish. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: app/code/Magento/Cms/view/adminhtml/ui_component/cms_block_listing.xml [hunk 1]
```diff
@@ -146,7 +146,6 @@
                 <editor>
                     <validation>
                         <rule name="required-entry" xsi:type="boolean">true</rule>
-                        <rule name="validate-xml-identifier" xsi:type="boolean">true</rule>
```
Comment (erfanimani) [inline app/code/Magento/Cms/view/adminhtml/ui_component/cms_block_listing.xml:149]: What server-side validation is being used? Just required-entry? Just making sure frontend is consistent with serverside validation

Comment (niravkrish) [reply app/code/Magento/Cms/view/adminhtml/ui_component/cms_block_listing.xml:149]: Hello @erfanimani ,

As Per Block add/edit in details page there is same validation.
That's Why i am removing another validation and add only require validation.

Please check attached screen shot
https://nimb.ws/VNP5Rz
https://nimb.ws/Rkzf2A

There is no validation for special character space and any other. there is only required validation is available.

Comment (niravkrish) [reply app/code/Magento/Cms/view/adminhtml/ui_component/cms_block_listing.xml:149]: Hello @erfanimani ,

As Per Block add/edit in details page there is same validation.
That's Why i am removing another validation and add only require validation.

Please check attached screen shot
https://nimb.ws/VNP5Rz
https://nimb.ws/Rkzf2A

There is no validation for special character space and any other. there is only required validation is available.

Comment (erfanimani) [reply app/code/Magento/Cms/view/adminhtml/ui_component/cms_block_listing.xml:149]: Perfect — I was just checking :)

Thank you

**Comment by rogyar** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @rogyar. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @rogyar, here is your [new Magento instance](https://pr-21376.instances.magento-community.engineering).
Admin access: [https://pr-21376.instances.magento-community.engineering/admin](https://pr-21376.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by niravkrish** [comment]
Hi @rogyar,
Please review and process this pull request.

**Comment by rogyar** [comment]
Hi @niravkrish. We need to perform some internal checks regarding the frontend/backend validation. In short, we need to ensure that the absence of backend validation is a correct implementation

**Comment by niravkrish** [comment]
Hi @rogyar,
Please review and process this pull request.

**Comment by rogyar** [comment]
For QA.
I've just double checked the possibility to use CMS blocks identifiers with spaces in layout and other places (widgets). There's no need to have "xml-identifier" validation (we don't have it on the backend side anymore) since the additional characters are allowed and blocks with identifiers that contain spaces and other characters work without issues.

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-4521](https://jira.corp.magento.com/browse/ENGCOM-4521) has been created to process this Pull Request

**Comment by soleksii** [comment]
:heavy_check_mark: QA Passed

**Result:**

after

**Pull request merged**